Bio/Entrez/Parser.py

```diff
@@ -434,8 +434,6 @@ def startNamespaceDeclHandler(self, prefix, uri):
             # to find out their first and last invocation for each namespace.
             self.namespace_level[prefix] += 1
             self.namespace_prefix[uri] = prefix
-            assert uri == "http://www.w3.org/1998/Math/MathML"
-            assert prefix == "mml"
```

Comment:
I guess this is still true in both cases?

Reply:
No, the case i tried i find that these are creating an unhandled error and stoping the parsing. Please take a look at this [example code](https://gist.github.com/Dhruv-Aga/c6f6f47cf4fd5f7a7bad78622f5a4709#file-issue-code-py) mentioned in  #3288. If there is any better solution, please guide me I will update my Pull request.

Reply:
I would suggest then to use
```python
    def startNamespaceDeclHandler(self, prefix, uri):
        """Handle start of an XML namespace declaration."""
        if prefix == "xsi":
            # This is an xml schema
            self.schema_namespace = uri
            self.parser.StartElementHandler = self.schemaHandler
        else:
            # Note that the DTD for MathML and XLink specify a default attribute
            # that declares the namespace for each MathML or Xlink element. This
            # means that these elements in the XML have an invisible namespace
            # declaration that triggers a call to startNamespaceDeclHandler
            # and endNamespaceDeclHandler. Therefore we need to count how often
            # startNamespaceDeclHandler and endNamespaceDeclHandler were called
            # to find out their first and last invocation for each namespace.
            if prefix == "mml":
                assert uri == "http://www.w3.org/1998/Math/MathML"
            elif prefix == "xlink":
                assert uri == "http://www.w3.org/1999/xlink"
            self.namespace_level[prefix] += 1
            self.namespace_prefix[uri] = prefix
        else:
            raise ValueError("Unknown prefix '%s' with uri %s" % (prefix, uri))

```

Reply:
Thanks, I will update my Pull Request